src/connectivity.jl

```diff
@@ -733,3 +733,66 @@ function isgraphical(degs::Vector{<:Integer})
     end
     return true
 end
+
+
+"""
+    isdigraphical(indegree_sequence, outdegree_sequence)
+
+Return true if the degree sequences `indegree_sequence` and `outdegree_sequence` satisfy the (simple) digraph realization problem.
+
+Two sequences of integers of integers satisfy the (simple) digraph realization problem, if there exists a simple directed graph where the indegrees of its vertices form that first sequence and the outdegrees the second.
+
+
+### Implementation Notes
+According to Fulkerson–Chen–Anstee theorem, a sequence ``\\{(a_1, b_1), ...,(a_n, b_n)\\}`` (sorted in descending order of a) is graphic iff the sum of vertex degrees is even and  ``\\sum_{i = 1}^{n} a_i = \\sum_{i = 1}^{n} b_i\\}`` and the sequence obeys the property -
+```math
+\\sum_{i=1}^{r} a_i \\leq \\sum_{i=r+1}^n min(r-1,b_i) + \\sum_{i=r+1}^n min(r,b_i)
+```
+for each integer r <= n-1
+"""
+function isdigraphical(indegree_sequence::Vector{<:Integer}, outdegree_sequence::Vector{<:Integer} )
+
+    n = length(indegree_sequence)
+
+    n == length(outdegree_sequence) || return false 
+
+    sum(indegree_sequence) == sum(outdegree_sequence) || return false
+
+    _sortperm = sortperm(indegree_sequence, rev = true)
+    
+    sorted_indegree_sequence = indegree_sequence[_sortperm]
+    sorted_outdegree_sequence = outdegree_sequence[_sortperm]
+
+    indegree_sum = zero(Int64)
+    outdegree_min_sum = zero(Int64)
+
+    cum_min = zero(Int64)
+
+    # The following approach, which requires substituting the line
+    # cum_min = sum([min(sorted_outdegree_sequence[i], r) for i in (1+r):n])
+    # with the line
+    # cum_min -= mindeg[r]
+    # inside the for loop below, work as well, but the values of `cum_min` at each iteration differ. To be on the safe side we implemented it as in https://en.wikipedia.org/wiki/Fulkerson%E2%80%93Chen%E2%80%93Anstee_theorem
+#=     mindeg = Vector{Int64}(undef, n)
+    @inbounds for i = 1:n
+        mindeg[i] = min(i, sorted_outdegree_sequence[i])
+    end
+    cum_min = sum(mindeg) =#
+    # Similarly for `outdegree_min_sum`.
+
+    @inbounds for r = 1:(n - 1)
+
+        indegree_sum += sorted_indegree_sequence[r]
+        outdegree_min_sum = sum([min(sorted_outdegree_sequence[i], r-1) for i in 1:r])
```

Comment:
Here we allocate a new Vector in each iteration of the loop. Can we do this differently?

Reply:
We were not able to use the same strategy adopted in `isgraphical`. For further details see in the comments.